input.go

```diff
@@ -24,41 +24,108 @@ type AlertSourceServiceDeleteInput struct {
-	TenantId              *string   `json:"tenantId,omitempty" yaml:"tenantId,omitempty" example:"Z2lkOi8vc2VydmljZS8xMjM0NTY3ODk"`             // The tenant OpsLevel uses to access the Azure account. (Optional.)
-	SubscriptionId        *string   `json:"subscriptionId,omitempty" yaml:"subscriptionId,omitempty" example:"Z2lkOi8vc2VydmljZS8xMjM0NTY3ODk"` // The subscription OpsLevel uses to access the Azure account. (Optional.)
-	ClientId              *string   `json:"clientId,omitempty" yaml:"clientId,omitempty" example:"Z2lkOi8vc2VydmljZS8xMjM0NTY3ODk"`             // The client OpsLevel uses to access the Azure account. (Optional.)
-	ClientSecret          *string   `json:"clientSecret,omitempty" yaml:"clientSecret,omitempty" example:""`                                    // The client secret OpsLevel uses to access the Azure account. (Optional.)
-	TagsOverrideOwnership *bool     `json:"tagsOverrideOwnership,omitempty" yaml:"tagsOverrideOwnership,omitempty" example:"false"`             // Allow tags imported from Azure to override ownership set in OpsLevel directly. (Optional.)
+	ClientId              *Nullable[string]   `json:"clientId,omitempty" yaml:"clientId,omitempty" example:"example_value"`                            // The client OpsLevel uses to access the Azure account. (Optional.)
+	ClientSecret          *Nullable[string]   `json:"clientSecret,omitempty" yaml:"clientSecret,omitempty" example:"example_value"`                    // The client secret OpsLevel uses to access the Azure account. (Optional.)
+	Name                  *Nullable[string]   `json:"name,omitempty" yaml:"name,omitempty" example:"example_value"`                                    // The name of the integration. (Optional.)
+	OwnershipTagKeys      *Nullable[[]string] `json:"ownershipTagKeys,omitempty" yaml:"ownershipTagKeys,omitempty" example:"['tag_key1', 'tag_key2']"` // An array of tag keys used to associate ownership from an integration. Max 5. (Optional.)
+	SubscriptionId        *Nullable[string]   `json:"subscriptionId,omitempty" yaml:"subscriptionId,omitempty" example:"example_value"`                // The subscription OpsLevel uses to access the Azure account. (Optional.)
+	TagsOverrideOwnership *Nullable[bool]     `json:"tagsOverrideOwnership,omitempty" yaml:"tagsOverrideOwnership,omitempty" example:"false"`          // Allow tags imported from Azure to override ownership set in OpsLevel directly. (Optional.)
+	TenantId              *Nullable[string]   `json:"tenantId,omitempty" yaml:"tenantId,omitempty" example:"example_value"`                            // The tenant OpsLevel uses to access the Azure account. (Optional.)
+}
+
+// CampaignCreateInput specifies the input fields used to create a campaign.
+type CampaignCreateInput struct {
+	CheckIdsToCopy *Nullable[[]ID]                  `json:"checkIdsToCopy,omitempty" yaml:"checkIdsToCopy,omitempty" example:"[]"`                  // The IDs of the existing rubric checks to be copied. (Optional.)
```

Comment:
I think this might be an edgecase of not really truely being "optional". Also i'm not sure nullable of a slice makes sense.

Reply:
This is definitely an odd case, but from a purely JSON perspective it makes sense. Unless some JSON schema forbids it, `null` fits pretty much anywhere. The mystery then is how it's handled on the app side